agent/php_user_instrument.c

```diff
@@ -515,15 +431,28 @@ void nr_php_reset_user_instrumentation(void) {
   nruserfn_t* p = nr_wrapped_user_functions;
 
   while (0 != p) {
+#if ZEND_MODULE_API_NO >= ZEND_7_4_X_API_NO
+    p->zf = NULL;
+#endif
     p->is_wrapped = 0;
     p = p->next;
   }
+#if ZEND_MODULE_API_NO >= ZEND_7_4_X_API_NO
+  if (NULL != user_function_wrappers) {
```

Comment:
Create a supportability metric with number of wraprecs for named user functions. What should be the name of the metric?

Reply:
`Supportability/PHP/Instrumented/Named` ?

Reply:
How about `Supportability/PHP/Instrumented/UserFunctions/Auto` for wraprecs created automatically by an agent (by calls to `nr_php_wrap_user_function` in `{fw|lib}_*_enable)`, `Supportability/PHP/Instrumented/UserFunctions/Transient` for wraprecs created 'in-flight' by intercepting `call_user_function` (WordPress hooks, Drupal hooks), and `Supportability/PHP/Instrumented/UserFunctions/Custom` for wraprecs created from PHP with call to [newrelic_add_custom_tracer](https://docs.newrelic.com/docs/apm/agents/php-agent/php-agent-api/newrelic_add_custom_tracer)? This opens up `Supportability/PHP/Instrumented/InternalFunctions/*` metric namespace in case we needed to count those too.

Reply:
LGTM

Reply:
This will be added in a separate PR.